OptimizelySDK.Tests/EventTests/EventBuilderTest.cs

```diff
@@ -241,8 +241,12 @@ public void TestCreateConversionEventNoAttributesNoValue()
                 {
                     { "Content-Type", "application/json"}
                 });
+            var experimentToVariationMap = new Dictionary<string, Variation>
```

Comment:
Can you clarify why this mapping was introduced with the bucketing ID implementation? I'm a little confused by it. Thanks!

Reply:
@alda-optimizely the problem is we fixed an issue in another pull request and at the same time we sent this pull request over the fix, that's why it's showing another pull request change. Here's the PR number https://github.com/optimizely/csharp-sdk/pull/22/files
Do I need to update this PR for reflecting only the bucketing ID change?

Reply:
no, but try to keep them separate next time if possible. thanks!

Reply:
Sure.